malaria-ios/Managers/GeneralAchievementsManager.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+/// Class that handles all the achievements related to the Rapid Fire Game.
+
+class GeneralAchievementsManager: NSObject {
```

Comment:
I don't follow this. In your description you describe "RapidFireAchievementsManager", "MedicineAchievementsManager", but it seems that this handles Trips. If so, rename the class...